Dockerfile

```diff
@@ -73,14 +78,22 @@ v${OSTICKET_VERSION}/osTicket-v${OSTICKET_VERSION}.zip; \
     cp -al /usr/local/src/osticket/. /var/www/html; \
     # Hide setup
     rm -r /var/www/html/setup; \
+    # Patch usage of GLOB_BRACE
+    apk add --no-cache --virtual .patch-deps patch; \
+    patch --directory="/var/www/html" --forward --strip=1 --input="/osticket-glob-brace.patch"; \
+    apk del .patch-deps; \
+    rm -rf /var/cache/apk/* /osticket-glob-brace.patch; \
```

Comment:
I'd be nice to start the script with build dependency installation and perform the removal in the end, just like the `RUN` above does it. That way, no refactoring or redundant `apk add` calls will be necessary if more build dependencies are added. Also, I'd move the actual patching and patch removal below `mv upload /usr/local/src/osticket`, as patching can be considered source preparation.

Reply:
I've re-ordered these steps as you suggested.